Gordon360/ApiControllers/AccountsController.cs

```diff
@@ -410,20 +430,20 @@ public IHttpActionResult AdvancedPeopleSearch(bool includeAlumniSearchParam, str
             {
                 if (!includeAlumniSearchParam)
                 {

```

Comment:
Maybe adding a  comment here would be better just for practice, for consistency and for a better code review